### PR TITLE
Misc Fixes

### LINKED ISSUE #12128: You can't unbuckle people from pipes
You can buckle a handcuffed person to pipes, and both they and you will have no way to get them free from said pipe unless you remove their handcuffs,

### LINKED ISSUE #12146: Xeno larvas now get flipped on their side when resting or knocked down/dead.
What it says on the tin, they're supposed to be lying on the ground, but not they're pointing up. Probably an easy sprite rotation fix.

Also I want to point out that they can flip either left or right, like humans can. Xenos don't do this normally.

### LINKED ISSUE #12193: Can't unbuckle from janicart using 'buckled' icon without first removing trash bag
You can using 'resist' though

Just a little consistency/QoL request

### PR DESCRIPTION
**Pull request by AnturK**
Fixes #12128 Hopefully not breaking anything..
Fixes #12146 
Fixes #12193 
Makes doors open when bumped in janicart.

### CODE REVIEW

File: code/game/machinery/machinery.dm [hunk 1]
```diff
@@ -284,6 +284,7 @@ Class Procs:
 
 //set_machine must be 0 if clicking the machinery doesn't bring up a dialog
 /obj/machinery/attack_hand(mob/user, check_power = 1, set_machine = 1)
+	..() // unbuckling etc
```
Comment (duncathan) [inline code/game/machinery/machinery.dm]: in most places like this, iirc, it's more like `if(..()) return 1`
maybe you should look at other unbuckling stuff to make sure? I could be wrong

Comment (AnturK) [inline code/game/machinery/machinery.dm]: Here i want to just call the side effects though. Not change the functionality.

Comment (duncathan) [inline code/game/machinery/machinery.dm]: well I'm pretty sure it's so that you don't unbuckle someone and activate a machine with one single click

**Comment by freerealestate** [comment]
>Makes doors open when bumped in janicart.

thank god

**Pull request merged**